src/v2/controllers/FormController.js

```diff
@@ -269,15 +277,16 @@ export default Controller.extend({
       Util.logConsoleError(error);
       errorObj = { responseJSON: { errorSummary: loc('error.unsupported.response', 'login')}};
     }
-    const showErrorBanner = !form?.isErrorCalloutCustomized(errorObj);
-    // show error before updating app state.
-    model.trigger('error', model, errorObj, showErrorBanner);
-    idxStateError = Object.assign({}, idxStateError, {hasFormError: true});
 
-    if (!showErrorBanner) {
-      form.showCustomErrorCallout(errorObj);
+    if(_.isFunction(form?.showCustomFormErrorCallout)) {
+      form.showCustomFormErrorCallout(errorObj);
+      this.showErrorBanner = !!form.showErrorBanner;
```

Comment:
shouldn't this be `false` if we just rendered a custom form error in the line above? Otherwise we will end up with more error callouts than what we need

Reply:
I implemented differently in latest commit.. but `!!` returns false for `false` and `undefined`